overlay/usr/lib/bitkey.d/inithooks-preseed

```diff
@@ -7,5 +7,6 @@ export ROOT_PASS=$MASTERPASS
 export USER_PASS=$MASTERPASS
 export SEC_UPDATES=SKIP
 export HUB_APIKEY=SKIP
+export SEC_ALERTS=SKIP
```

Comment:
The security updates will still install in a live environment fine. If we had to re-release BitKey every time Debian release a security update we'll need to release every few days!

Although you do raise a good point. Perhaps we could consider some mechanism that would support the user updating the Live ISO themselves? #16